tests/fast-integration/monitoring/prometheus.js

```diff
@@ -84,29 +84,36 @@ async function assertAllRulesAreHealthy() {
 }
 
 async function assertMetricsExist() {
-    await assertTimeSeriesExist("kube_deployment_status_replicas_available", [
-        "deployment",
-        "namespace",
-    ]);
-    await assertTimeSeriesExist("istio_requests_total", [
-        "destination_service",
-        "response_code",
-        "source_workload",
-    ]);
-    await assertTimeSeriesExist("container_memory_usage_bytes", [
-        "pod",
-        "container",
-    ]);
-    await assertTimeSeriesExist(
-        "kube_pod_container_resource_limits_memory_bytes",
-        ["pod", "container"]
-    );
-    await assertTimeSeriesExist("container_cpu_usage_seconds_total", [
-        "container",
-        "pod",
-        "namespace",
-    ]);
-    await assertTimeSeriesExist("kube_service_labels", ["namespace"]);
+    let metricsList = [
+        {"kubelet":[
+                {"container_memory_usage_bytes": ["pod","container"]},
+                {"kubelet_volume_stats_available_bytes": []},
+                {"kubelet_pod_start_duration_seconds_count": []}]},
+
+        {"api-server":[
+                {"apiserver_request_total": []},
+                {"apiserver_request_duration_seconds_bucket": []},
+                {"etcd_disk_backend_commit_duration_seconds_bucket": []}]},
+
+        {"kube-state-metrics":[
+                {"kube_deployment_status_replicas_available": ["deployment","namespace"]},
+                {"kube_pod_info": ["pod"]},
+                {"kube_pod_container_resource_limits_memory_bytes": ["pod","container"]}]},
```

Comment:
Should be adapted once https://github.com/kyma-project/kyma/pull/13037 is merged (please check the PR description, the metric is renamed)